src/cow-react/utils/fractionUtils.ts

```diff
@@ -17,4 +19,34 @@ export class FractionUtils {
       return null
     }
   }
+
+  static fractionLikeToExact(amount: Nullish<FractionLike>, max = FULL_PRICE_PRECISION): string {
```

Comment:
No need to change anything, just a question to understand your approach: why have you chosen to use a static class rather than assorted functions?

Reply:
1. To have all utils in one scope (for me it helps to understand why we use these utils for)
2. We don't need the utils to be tree shaken, because we use them all